War Engines first pass (#28)
GameBrains/er-core: modules/special-rules/pages/war-engines-in-firefights.adoc

War engines participate in firefights in much the same way as other units.

> * Count the Firepower of each weapon system that has at least one enemy unit within fire arc and within 15 cm.
> //IJW editing note: firefights aren't restricted by line of fire, so this has to refer to the weapon arc instead.

Reviewer: Ah yes, good catch. Just a tweak to correct 'fire arc' to 'weapon arc' to be consistent with our definition.
```suggestion
* Count the Firepower of each weapon system that has at least one enemy unit within its weapon arc and within 15 cm.
```
